apps/controllerx/cx_devices/livarno.py

```diff
@@ -0,0 +1,19 @@
+from cx_const import Light, TypeActionsMapping
+from cx_core import LightController
+
+
+class HG06323Controller(LightController):
```

Comment:
Could you change `HG06323Controller` to `HG06323LightController`. I try to state the type of controller in the class name for new devices.